signalbot/__init__.py

```diff
@@ -1,5 +1,5 @@
 from .bot import SignalBot
-from .command import Command, CommandError, triggered
+from .command import Command, CommandError, triggered, regex_triggered
```

Comment:
Could you also modify the command section in the Readme to mention the `regex_triggered` next to the `triggered`?

Reply:
Got it :+1: